src/components/ContractWriteResultCard/ContractWriteResultCard_View.tsx

```diff
@@ -1,16 +1,23 @@
 import { shortenAddress } from "../../utils/formatting";
+import { TxType } from "../Vault/Vault_Logic";
 
 type Props = {
   hash?: string;
   isSuccess: boolean;
   errorMsg?: string;
+  txType: TxType.BET | TxType.DEPOSIT | TxType.WITHDRAWAL | undefined;
```

Comment:
if `TxType` is an enum you can just assign `txType` to it -- also if its optionally `undefined` it can be written as `txType?: TxType`